.github/workflows/build.yml

```diff
@@ -47,4 +47,6 @@ jobs:
           Xvfb -ac :99 -screen 0 1920x1080x16 &
           gradle runIdeForUiTests &
       - name: Run the UI tests
+        env:
+          APLUS_TEST_TOKEN: ${{ secrets.APLUS_TEST_TOKEN }}
```

Comment:
I suggest thinking carefully about the credentials used for UI testing, as we don't have many pairs of them, and the one used in automation will get polluted very very quickly (once we get to submissions) @Aalto-LeTech/apluscourses-plugin-team